.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# What does this PR do ?
+
+Add a one line overview of what this PR aims to accomplish.
+
+# Changelog
+
+- Add specific line by line info of high level changes in this PR.
+
+# Usage
+
+- You can potentially add a usage example below
+
+```python
+# Add a code snippet demonstrating how to use this
+```
+
+# GitHub Actions CI
+
+See the [CI section](https://github.com/NVIDIA-NeMo/Megatron-Bridge/blob/main/CONTRIBUTING.md#-running-github-ci)in the Contributing doc for how to trigger the CI. A Nvidia developer will need to approve and trigger the CI for external contributors.
+
+# Before your PR is "Ready for review"
+
+**Pre checks**:
+
+- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA-NeMo/Megatron-Bridge/blob/main/CONTRIBUTING.md)
+- [ ] Did you write any new necessary tests?
+- [ ] Did you add or update any necessary documentation?
+- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
+  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
+
+**PR Type**:
+
+- [ ] New Feature
+- [ ] Bugfix
+- [ ] Documentation
```

Comment:
don't think this is necessary either - it might be redundant with the labels. it should also be clearer from the linked github issue which PR type this is